projects/update/client/updateProject.js

```diff
@@ -0,0 +1,58 @@
+Template.updateProject.rendered = function() {
+  // Get reference to template instance
+  let instance = this;
+
+  instance.autorun(function () {
+    // Make sure tags subscription is ready
+    if (instance.subscriptionsReady()) {
+      // Get all existing tags
+      let tagOptions = Tags.find().fetch();
+
+      $('#tags').selectize({
+        delimiter: ',',
+        persist: false,
+        valueField: 'name',
+        labelField: 'name',
+        searchField: 'name',
+        highlight: true,
+        maxOptions: 5,
+        options: tagOptions,
+        create:true,
+        onItemAdd(value){
+          // Insert tag into Tags collection if it doesn't exist
+          if(!Tags.findOne({"name":value})){
+            Tags.insert({"name":value});
+          }
+        }
+      });
+    }
+  });
+};
+
+Template.updateProject.helpers({
+    //Using the Flowrouter package to get the route param projects/:projectId ; is set up in the Template.updateProject.onCreated callback
+    getProject: function () {
```

Comment:
Lets shorten this to `projects` for brevity. This would also be consistent with our other template helpers, which tend to be nouns - as opposed to the current verb phrase `get project`.